BraintreeCore/src/main/java/com/braintreepayments/api/core/AnalyticsClient.kt

```diff
@@ -74,10 +78,11 @@ internal class AnalyticsClient(
         authorization: Authorization,
         integration: IntegrationType?
     ): UUID {
+        val sessionId = analyticsParamRepository.sessionId
```

Comment:
If the issue was that this value might change do we need to make a copy of it here or is this enough to make it immutable in the inputData?

Reply:
It's a very small edge case where this value can change in between creating the input data and enqueuing the work. But yep, creating a copy here ensures that the session ID in the `uniqueWorkName` and the session ID in the inputData match.